src/component/chrome/chrome-container.tsx

```diff
@@ -24,13 +27,17 @@ export class ChromeContainer extends React.Component {
 		}
 
 		return (
-			<Chrome
-				leftVisible={!!previousPage}
-				rightVisible={!!nextPage}
-				onLeftClick={() => (previousPage ? store.openPage(previousPage.getId()) : undefined)}
-				onRightClick={() => (nextPage ? store.openPage(nextPage.getId()) : undefined)}
-				title={page ? page.getName() : undefined}
-			>
+			<Chrome>
+				<OverviewSwitchContainer />
+				<ViewSwitch
+					fontSize={Size.M}
+					justify="center"
+					leftVisible={!!previousPage}
+					rightVisible={!!nextPage}
+					onLeftClick={() => (previousPage ? store.openPage(previousPage.getId()) : undefined)}
```

Comment:
`onLeftClick` is not optional. It should not be valid to pass undefined as a value here.
Maybe just always pass the function. I will not be triggered because the buttons are not visible if not needed

Reply:
Done ✅